### ISSUE TITLE
Ephemeral_Auth: DN path length greater than 129 characters causes exception

### ISSUE THREAD

**Issue by billchurch** [billchurch/f5-pua#22]
User attempts to open (login) a Webtop Portal Access Resource and the users DN path length is greater than 129 characters, the script fails with an error of:
 
```
err tmm[18953]: 01220001:3: TCL error: /Common/ephemeral_auth_plugin/ldap_proxy_ssl <CLIENTSSL_DATA> - bad field specifier "-"     while executing "binary scan $protocolData x[expr {$startbytes}]A[expr {$bindDN_len}] bindDN"    (iRule proc "/Common/ephemeral_auth_plugin/ldap_proxy_ssl::parse_bindRequest") (line 23)     invoked from within "call parse_bindRequest [lindex $result 2]"     ("bindRequest" arm line 6)     invoked from within "switch $protocolOp {       "bindRequest" {         set ephemeral_LDAP_Bypass 0 log local0. ""  log local0. "RESULT: $result" log local0. ""         se..."
 ```

If you move that account to an OU with a pat of 128 or less, no issues.
 
                (ISSUE / 136 Characters)
                CN=svc.abcd.f5_test_use,OU=Service Accounts,OU=Big-IP F5,OU=Services,OU=ORGB,OU=Place A,OU=Installations,OU=abcd,OU=orga,DC=ab,DC=local
                (NO ISSUES / 128 Characters)
                CN=svc.abcd.f5_test_use,OU=ServiceA,OU=Big-IP F5,OU=Services,OU=ORGB,OU=Place A,OU=Installations,OU=abcs,OU=orga,DC=ab,DC=local

**Comment by billchurch** [comment]
Should accept characters as long as an LDAP client can create/bind with. The theoretical maximum for an AD LDAP bind is 255. Tested bind against both ldap_proxy and ldaps_proxy with a DN up to 463 characters.

**Comment by billchurch** [comment]
fixed in (f045797)
